march_simulation/obstacles/ramp_and_door.xacro

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0"?>
+<robot name="ramp_and_door" xmlns:xacro="http://www.ros.org/wiki/xacro">
+    <gazebo>
+        <static>true</static>
+    </gazebo>
+
+
+    <!-- Sizes from the rules of the cybathlon 2020 -->
+    <xacro:property name="platform_height" default="0.48"/>
+    <xacro:property name="platform_length" default="2.920"/>
+    <xacro:property name="platform_width" default="1.4"/>
+    <xacro:property name="ramp_up_length" default="1.403"/>
+    <xacro:property name="ramp_down_length" default="1.854"/>
+    <xacro:property name="distance_to_origin" default="0.4"/>
+
+    <xacro:property name="parent_name" value="world"/>
+    <xacro:property name="mu1" value="1.0"/>
+    <xacro:property name="mu2" value="1.0"/>
+    <!-- Minimum allowable depth before contact correction impulse is applied. -->
+    <xacro:property name="min_depth" value="0.0005"/>
+
+    <!-- Used for fixing robot to Gazebo 'base_link' -->
+    <link name="${parent_name}"/>
+
+
+    <!-- Links -->
+    <link name="platform">
+        <collision>
+          <geometry>
+            <box size="${platform_length} ${platform_width} 3" />
```

Comment:
Why 3? Why not just platform_height?

Reply:
I did this to get it stabilized (or trying to) before we solve the fixing problem. I have changed it now to platform height.